examples/imitation_learning_openbot/run_trained_policy.py

```diff
@@ -114,69 +101,47 @@
 
             # change config based on current scene
             config.defrost()
-
-            if episode["scene_id"] == "kujiale_0000":
-                config.SIMULATION_CONTROLLER.SCENE_ID = episode["scene_id"]
-                config.SIMULATION_CONTROLLER.MAP_ID   = episode["scene_id"] + rendering_mode_map_str
-
-                # kujiale_0000 has scene-specific config values
-                scene_config_file = os.path.realpath(os.path.join(os.path.dirname(__file__), "scene_config.kujiale_0000.yaml"))
-
-            elif episode["scene_id"] == "warehouse_0000":
-                config.SIMULATION_CONTROLLER.SCENE_ID = episode["scene_id"]
-                config.SIMULATION_CONTROLLER.MAP_ID   = episode["scene_id"]
-
-                # warehouse_0000 has scene-specific config values
-                scene_config_file = os.path.realpath(os.path.join(os.path.dirname(__file__), "scene_config.warehouse_0000.yaml"))
-
-            else:
-                assert False
-                
-            config.merge_from_file(scene_config_file)
+            config.SIMULATION_CONTROLLER.SCENE_ID = episode["scene_id"]
             config.freeze()
 
             # create Env object
-            env = spear.Env(config=config)
+            env = OpenBotEnv(config=config)
 
         # reset the simulation
         env_reset_info = {}
-        _ = env.reset(reset_info=env_reset_info)
+        obs = env.reset(reset_info=env_reset_info)
         assert "success" in env_reset_info
 
-        # if it took too long to reset the simulation, then continue
-        if not env_reset_info["success"]:
-            spear.log("Call to env.reset(...) was not successful. Simulation took too long to return to a ready state. Skipping...")
-            prev_scene_id = episode["scene_id"]
-            continue
-
-        # send zero action to the agent and collect initial trajectory observations:
-        obs, _, _, env_step_info = env.step(action={"apply_voltage": np.array([0.0, 0.0], dtype=np.float32)})
+        # get a trajectory for this episode based on start and end point
+        episode_start_location = [[episode["start_location_x"], episode["start_location_y"], episode["start_location_z"]]]
+        episode_goal_location  = [[episode["goal_location_x"], episode["goal_location_y"], episode["goal_location_z"]]]
+        trajectory = env.get_trajectories(episode_start_location, episode_goal_location)
 
```

Comment:
Add a `policy.reset(goal)` function here to pass in the high-level conditioning data (the goal position), similar to how we do it in `OpenBotPathFollowingPolicy`

Reply:
Updated this